walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,120 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.HashSet;
+import java.util.List;
 
 public final class FindMeetingQuery {
+
+  /**
+   * Finds time slots available to host a meeting given the attendees's events and a meeting duration.
+   *
+   * Algorithmic complexity: Time O(mn), Space O(m) 
+   * - First it checks for every event (n) if there's at least one attendee (n) from the request. In the worst case it no event will have such attendee.
```

Comment:
nit: second sentence (In the worst case...), remove "it"? not sure

Reply:
Yes, I didn't understand that either haha, changed.